src/test/resources/network/brightspots/rcv/test_data/test_set_only_one_winner_per_round/test_set_only_one_winner_per_round_config.json

```diff
@@ -0,0 +1,57 @@
+{
+  "outputSettings" : {
+    "contestName" : "One Winner per Round Test",
+    "outputDirectory" : "output",
+    "contestDate" : "2019-01-18",
+    "contestJurisdiction" : "",
+    "contestOffice" : "",
+    "tabulateByPrecinct" : false,
+    "generateCdfJson" : false
+  },
+  "cvrFileSources" : [ {
+    "filePath" : "/Users/leisenberg/code/rcv/src/test/resources/network/brightspots/rcv/test_data/test_set_only_one_winner_per_round/test_set_only_one_winner_per_round_cvr.xlsx",
```

Comment:
Bad file path in here; it's hard-coded to your system.

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/096762b1-9614-405f-9b03-f3accc60ffdd-0878c302-d67c-4bb7-80ba-b554896c3f48-v1.png?transparent=1&palette=1&width=246)